packages/core/admin/admin/src/components/Theme/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { ThemeProvider } from 'styled-components';
 import PropTypes from 'prop-types';
-import GlobalStyle from '../GlobalStyle';
+// import GlobalStyle from '../GlobalStyle';
```

Comment:
ℹ️ In my PR, I've left this import here but removed everything in the template literals. The reason is that the GlobalStyles import fontawesome and that we don't have a real plan for removing it for now 😅